### DISCUSSION TITLE
Null-ls code actions not working

### DISCUSSION THREAD

**Discussion by uldrim** [LazyVim/LazyVim#969] [Q&A]
I have been using LazVim and I configured it for typescript.
But code actions are not working in my fresh install.
Any help is appreciated.
These are my config files
![Screenshot from 2023-06-24 15-48-57](https://github.com/LazyVim/LazyVim/assets/45782936/e1fd19fb-4110-4b89-97d8-21b67ec96c80)
![Screenshot from 2023-06-24 15-48-25](https://github.com/LazyVim/LazyVim/assets/45782936/036f8556-d9b0-4215-80b0-b502f3ecd315)
![Screenshot from 2023-06-24 15-47-50](https://github.com/LazyVim/LazyVim/assets/45782936/a021c68b-50f5-4b98-ac58-d4b7ed411d9d)

**Comment by folke** [comment]
You're probably overwriting the null-ls sources in your config.

`sources` is a list so that can't me just merged, you need to manually update that to make it work properly.

This is that the typescript extra does:

```lua
  {
    "jose-elias-alvarez/null-ls.nvim",
    opts = function(_, opts)
      table.insert(opts.sources, require("typescript.extensions.null-ls.code-actions"))
    end,
  },

**Reply by uldrim** [reply]
Thanks for the reply.
But, I did not change anything.
I simply cloned the repo, hit nvim, and after that I uncommented three lines in lazy.lua file
I followed this video https://www.youtube.com/watch?v=N93cTbtLCIM&t=185s&ab_channel=ElijahManor
I have been using LazyVim for a few months, never had any such issue. I don't know what I did wrong in this install.
